### PR TITLE
DDP-4913: Prion migration branch

### PR DESCRIPTION
**Pull request by charlottedye**
In order to prevent automated deployments from removing the Prion code from dev (https://prion-dot-broad-ddp-dev.appspot.com/), I'd like to merge my Prion branches into develop.  The Prion migration is not yet complete.  I don't expect anything in this branch to cause any changes to other studies, so in that sense this is low risk, but it's high risk in that there is a lot of code here.  Please also refer to the frontend pull request here: https://github.com/broadinstitute/ddp-angular/pull/261

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [ ] HELP WANTED: I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] HELP WANTED: If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [x] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [ ] :relaxed: All good, business as usual!
- [x] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [x] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [x] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: elasticsearch/roles/prion_index_read_phi.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (yufengwng) [inline elasticsearch/roles/prion_index_read_phi.json]: Good news! In the latest release, we have granted DSM full read access (`pepper_index_read_phi`) to data in Elasticsearch, so I don't think we'll need a separate role for grant read on prion data.

File: elasticsearch/roles/prion_kibana_admin.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (yufengwng) [inline elasticsearch/roles/prion_kibana_admin.json]: Honestly, we don't really expose Kibana for folks to use. I'll say to scrap this for now.

File: study-builder/tenants/prion/platform.en.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (yufengwng) [inline study-builder/tenants/prion/platform.en.json]: I don't think we need this file?

File: pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoaderMain.java [hunk 4]
```diff
@@ -347,6 +347,8 @@ public static void initDbConnection() {
         JsonElement combinedConsentSurveyData = surveyData.getAsJsonObject().get("combinedconsentsurvey");
         JsonElement followupSurveyData = surveyData.getAsJsonObject().get("followupsurvey");
         JsonElement followupConsentSurveyData = surveyData.getAsJsonObject().get("followupconsentsurvey");
+        JsonElement prionConsentSurveyData = surveyData.getAsJsonObject().get("ConsentSurvey");
+        JsonElement medicalSurveyData = surveyData.getAsJsonObject().get("MedicalSurvey");
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoaderMain.java:352]: I'm not too sure about study migration and loader code, but I hope this doesn't conflict with what we're doing for migration other studies. Please coordinate with @ssettipalli when he gets back.

**Review by yufengwng — COMMENTED** [review]
Looks good! There is a lot of changes here,  but they are all scoped to `studies/prion` so I think it's perfectly fine to merge this in.

**Comment by charlottedye** [comment]
Broken...  recreating

**Pull request closed without merging**